### PR TITLE
CB-13067:Add Entitlement for AWS CMK cli

### PR DESCRIPTION
**Pull request by satyabollineni**

### CODE REVIEW

**Comment by satyabollineni** [comment]
Associated Thunderhead PR: https://github.infra.cloudera.com/thunderhead/thunderhead/pull/5052

File: auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/EntitlementService.java [hunk 1]
```diff
@@ -270,6 +271,9 @@ public boolean isAzureDiskSSEWithCMKEnabled(String accountId) {
         return isEntitlementRegistered(accountId, CDP_CB_AZURE_DISK_SSE_WITH_CMK);
     }
 
+    public boolean isAWSDiskEncryptionWithCMKEnabled(String accountId) {
+        return  isEntitlementRegistered(accountId, CDP_CB_AWS_DISK_ENCRYPTION_WITH_CMK); }
```
Comment (juhi-09) [inline auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/EntitlementService.java:275]: Please fix the spacing (2 spaces after `return`) and move closing brace on new line.

**Review by juhi-09 — CHANGES_REQUESTED** [review]

File: auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/EntitlementService.java [hunk 2]
```diff
@@ -13,6 +13,7 @@
 import static com.sequenceiq.cloudbreak.auth.altus.model.Entitlement.CDP_AZURE_SINGLE_RESOURCE_GROUP_DEDICATED_STORAGE_ACCOUNT;
 import static com.sequenceiq.cloudbreak.auth.altus.model.Entitlement.CDP_BASE_IMAGE;
 import static com.sequenceiq.cloudbreak.auth.altus.model.Entitlement.CDP_CB_AZURE_DISK_SSE_WITH_CMK;
+import static com.sequenceiq.cloudbreak.auth.altus.model.Entitlement.CDP_CB_AWS_DISK_ENCRYPTION_WITH_CMK;
 import static com.sequenceiq.cloudbreak.auth.altus.model.Entitlement.CDP_CB_DATABASE_WIRE_ENCRYPTION;
```
Comment (lajosrodek) [inline auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/EntitlementService.java:17]: Wrong import order

File: mock-thunderhead/src/main/java/com/sequenceiq/thunderhead/grpc/service/auth/MockUserManagementService.java [hunk 3]
```diff
@@ -54,6 +54,7 @@
 import static com.sequenceiq.cloudbreak.auth.altus.model.Entitlement.FMS_FREEIPA_BATCH_CALL;
 import static com.sequenceiq.cloudbreak.auth.altus.model.Entitlement.LOCAL_DEV;
 import static com.sequenceiq.cloudbreak.auth.altus.model.Entitlement.CDP_CB_AZURE_DISK_SSE_WITH_CMK;
+import static com.sequenceiq.cloudbreak.auth.altus.model.Entitlement.CDP_CB_AWS_DISK_ENCRYPTION_WITH_CMK;
 import static com.sequenceiq.cloudbreak.auth.altus.model.Entitlement.OJDBC_TOKEN_DH;
```
Comment (lajosrodek) [inline mock-thunderhead/src/main/java/com/sequenceiq/thunderhead/grpc/service/auth/MockUserManagementService.java:58]: Wrong import order. (See other static imports as well; the whole `Entitlement.*` part seems to be messed up.)

File: auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/EntitlementService.java (hunk 1 above)
Comment (juhi-09) [reply auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/EntitlementService.java]: LGTM.

**Review by juhi-09 — APPROVED** [review]

**Review by lajosrodek — APPROVED** [review]
Thanks, LGTM

**Pull request merged**